terraform/terraform.tfvars

```diff
@@ -0,0 +1,20 @@
+# Mandatory Config Options
+model_name           = "put your model-name here"
+amf_application_name = "put your amf app name here"
+
+# Optional Configuration
+channel = "put the charm channel here"
+gnb-config = {
+  gnb-ip-address = "put GNB IP address  here"
+  gnb-interface  = "put GNB interface here"
+  imsi           = "put IMSI here"
+  mcc            = "put MCC here"
+  mnc            = "put MNC here"
+  usim-key       = "put USIM Key here"
+  usim-opc       = "put USIM OPC here"
+  sd             = "put SD here"
+  sst            = "put SST here"
+  tac            = "put TAC here"
+  upf-subnet     = "put UPF subnet here"
+  upf-gateway    = "put UPF gateway here"
+}
```

Comment:
Because those configurations are optional, is it possible to use the default values? I am not sure how this would be done.

Reply:
If we do not set them here, the default configurations are used which are already set in the charm.  I commented that options, so users do not need to delete anything. If they would  like to set, they can just uncomment the line and set it.